01week/rockPaperScissors.js

```diff
@@ -1,16 +1,41 @@
 'use strict';
-
+// i will
 const assert = require('assert');
 const readline = require('readline');
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
 
-
+// ready ready
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  // ROCK scissors PAPER
+ // 2 player game where each player's input is compared to one another and the value of their input determines who wins
+ //in the game rock is > then scissors but < paper
+ //paper is > rock but, sciccors
+//   sciccors is > paper but <rock
+// User1 input of ['rock', 'paper' , 'scissors'.]
+// User2 input of ['rock', 'paper', or 'scissors'.]
+// Compare User1 input to User2 input.
+
+let User1 = prompt ("Player numero1 rock, paper, scissors");
```

Comment:
Where is the prompt function defined?  Because this function is undefined, the variables of User1 and User2 are not being defined and tested in your conditionals.  

Also, don't use let here.

Reply:
The prompt is a built-in function for Javascript, allowing us to get the user input from the browser.  This allows us to define User1 and User2 values, per the comment below.

Reply:
What's the reason not to use Let?